webpack.config.js

```diff
@@ -35,7 +35,8 @@ const config = module.exports = {
     ]
   },
   entry: {
-    'metamaps.bundle': './frontend/src/index.js'
+    'metamaps.bundle': './frontend/src/index.js',
+    'homepage': './frontend/src/homepage.js',
```

Comment:
really happy to do it another way, I just wouldn't know how. Not familiar with webpack at all really